project.yaml

```
@@ -6,7 +6,7 @@ expectations:
 
 actions:
   generate_dataset:
-    run: ehrql:v1 generate-dataset analysis/dataset_definition.py --output output/dataset.csv.gz
+    run: ehrql:v1 generate-dataset analysis/dm_reg_dataset_atamborska.py --output output/dataset.csv.gz
     outputs:
       highly_sensitive:
         dataset: output/dataset.csv.gz
```

this will create a subdirectory for all the diabetes results and also a more descriptive file name
```suggestion
        dataset: output/dm/dm_reg_atamborska.csv.gz
```